src/modules/plus/filter_strobe.yml

```diff
@@ -0,0 +1,31 @@
+schema_version: 0.3
+type: filter
+identifier: alpha_strobe
+title: Alpha strobing
+version: 1
+copyright: Martin Sandsmark
+creator: Martin Sandsmark
+license: LGPLv2.1
+language: en
+description: Strobes the alpha channel to 0. Many other filters overwrite the alpha channel, in that case this needs to be last.
+tags:
+  - Video
+parameters:
+  - identifier: strobe_invert
+    title: Invert
+    type: boolean
+    description: Whether to invert which frames are on and which is off
+    default: false
```

Comment:
We do not use `true` or `false` as values for booleans in the service metadata. You can see the `metaschema.yml` for reference. Please change this to 0.